src/bone.js

```diff
@@ -371,15 +371,15 @@ class Bone {
    */
   previousChanges(name) {
     if (name != null) {
-      if (this.#rawUnset.has(name) || this.#rawPrevious[name] === undefined || !this.hasAttribute(name)) return {};
+      if (this.#rawUnset.has(name) || (this.#rawPrevious[name] === undefined && this.isNewRecord) || !this.hasAttribute(name)) return {};
```

Comment:
It looks like `this.isNewRecord` covers `this.#rawPrevious[name] === undefined` already, can we give it a try?

btw, this line is a bit too much now, I believe a multiple line if block is necessary once the column width exceeds some threshold like 90 or 120